bundles/org.openhab.binding.doorbird/src/main/java/org/openhab/binding/doorbird/internal/DoorbirdHandler.java

```diff
@@ -0,0 +1,622 @@
+            case CHANNEL_LIGHT:
+                handleLight(command);
+                break;
+            case CHANNEL_OPENDOOR1:
+                handleOpenDoor(command, "1");
+                break;
+            case CHANNEL_OPENDOOR2:
+                handleOpenDoor(command, "2");
+                break;
+            case CHANNEL_RESTART:
+                handleRestart(command);
+                break;
+            case CHANNEL_GETIMAGE:
+                handleGetImage(command);
+                break;
+            case CHANNEL_DOORBELL_HISTORY_INDEX:
+            case CHANNEL_MOTION_HISTORY_INDEX:
+                if (!(command instanceof RefreshType)) {
+                    handleHistoryImage(channelUID, command);
+                }
+                break;
+            case CHANNEL_GET_DOORBELL_MONTAGE:
+                if (command instanceof RefreshType || (command instanceof OnOffType && command.equals(OnOffType.ON))) {
+                    updateDoorbellMontage();
+                }
+                break;
+            case CHANNEL_GET_MOTION_MONTAGE:
+                if (command instanceof RefreshType || (command instanceof OnOffType && command.equals(OnOffType.ON))) {
+                    updateMotionMontage();
+                }
+                break;
+            case CHANNEL_SIP_HANGUP:
+                handleSipHangup(command);
+                break;
+        }
+    }
+
+    private void refreshDoorbellImageFromHistory() {
+        logger.debug("Handler: REFRESH doorbell image channel using most recent doorbell history image");
+        scheduler.execute(() -> {
+            String url = buildUrl("/bha-api/history.cgi", "?event=doorbell&index=1");
+            DoorbirdImage dbImage = downloadImage(url);
+            if (dbImage != null) {
+                RawType image = dbImage.getImage();
+                updateState(CHANNEL_DOORBELL_IMAGE, image != null ? image : UnDefType.UNDEF);
+                updateState(CHANNEL_DOORBELL_TIMESTAMP, getLocalDateTimeType(dbImage.getTimestamp()));
+            }
+            updateState(CHANNEL_DOORBELL, OnOffType.OFF);
+        });
+    }
+
+    private void refreshMotionImageFromHistory() {
+        logger.debug("Handler: REFRESH motion image channel using most recent motion history image");
+        scheduler.execute(() -> {
+            String url = buildUrl("/bha-api/history.cgi", "?event=motionsensor&index=1");
+            DoorbirdImage dbImage = downloadImage(url);
+            if (dbImage != null) {
+                RawType image = dbImage.getImage();
+                updateState(CHANNEL_MOTION_IMAGE, image != null ? image : UnDefType.UNDEF);
+                updateState(CHANNEL_MOTION_TIMESTAMP, getLocalDateTimeType(dbImage.getTimestamp()));
+            }
+            updateState(CHANNEL_MOTION, OnOffType.OFF);
+        });
+    }
+
+    private void handleLight(Command command) {
+        if (command instanceof OnOffType && command.equals(OnOffType.ON)) {
```

Comment:
Is it not possible to turn off the light?

Reply:
No. The device turns it off automatically after a defined amount of time.